### PR TITLE
Tweaks for bookshelf list view

### PR DESCRIPTION
**Pull request by benonymity**
Add lines to more easily distinguish between items in the app list view and add duration label.
List View

### CODE REVIEW

**Comment by advplyr** [comment]
Nice touch! Podcasts don't have a duration on the media object, so we can hide the duration for podcasts. Maybe we show number of episodes instead.

**Comment by benonymity** [comment]
Good idea! Duration should now be hidden for podcasts and episode numbers shown instead. 

Episode Numbers

File: components/cards/LazyListBookCard.vue [hunk 1]
```diff
@@ -99,9 +101,21 @@ export default {
     mediaType() {
       return this._libraryItem.mediaType
     },
+    duration() {
+      return this.media.duration || null
+    },
     isPodcast() {
       return this.mediaType === 'podcast'
     },
+    episodes() {
```
Comment (advplyr) [inline components/cards/LazyListBookCard.vue:110]: Computed property should always return some value

Comment (benonymity) [reply components/cards/LazyListBookCard.vue:110]: Whoops! I'll keep that in mind for the future--this is the first time I've used Vue so still learning the ropes.

**Pull request merged**